src/tacacs/audisp/Makefile

```diff
@@ -0,0 +1,30 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = audisp-tacplus_$(AUDISP_TACPLUS_VERSION)_$(CONFIGURED_ARCH).deb
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Obtain audisp-tacplus
+	rm -rf ./audisp-tacplus
+
+	git clone https://github.com/daveolson53/audisp-tacplus.git
+
+	# checkout by sha1
+	pushd ./audisp-tacplus
+	git checkout 559c9f22edd4f2dea0ecedffb3ad9502b12a75b6
+
+	# Apply patches
+	cp -r ../patches patches
+	quilt push -a
+
+	# fix aclocal depency issue by run autoconf
+	autoreconf
```

Comment:
>autoreconf [](http://example.com/codeflow?start=1&length=10)

Is it same as [https://github.com/daveolson53/audisp-tacplus/blob/master/auto.sh](https://github.com/daveolson53/audisp-tacplus/blob/master/auto.sh)?

Reply:
Yes, the original 'configure' file depends on a older version of aclocal, so we need run autoconf first to fix the aclocal version issue.

Reply:
Then use `./auto.sh` ?

Reply:
Fixed.